### PR TITLE
support priority class for arena

### PR DESCRIPTION
**Pull request by cheyang**
1. Submit the training job with priority
2. Check the Priority Class Name by using `get` 
  
- scenario 1: submit the training job(mpi and tfjob) by setting priority

```
arena submit mpi              \
--name=critical               \
--priority=critical           \
--gpus=1                    \
--workers=1                 \
--image=registry.aliyuncs.com/tensorflow-samples/horovod:0.13.11-tf1.10.0-torch0.4.0-py3.5                   "mpirun tail -f /dev/null"
configmap/critical-mpijob created
configmap/critical-mpijob labeled
mpijob.kubeflow.org/critical created
INFO[0001] The Job critical has been submitted successfully
INFO[0001] You can run `arena get critical --type mpijob` to check the job status
```

- scenario 2: the priority name doesn't exist, it will do pre-check

```
# arena submit mpi              \
--name=critical1               \
--priority=critical1           \
--gpus=1                    \
--workers=1                 \
--image=registry.aliyuncs.com/tensorflow-samples/horovod:0.13.11-tf1.10.0-torch0.4.0-py3.5       \            "mpirun tail -f /dev/null"
The priority critical1 doesn't exist. Please check with `kubectl get pc` to get validate priority name.
```

- scenario 3: if the priority is set, you can check it by using `arena get`

```
arena get critical
STATUS: RUNNING
NAMESPACE: default
PRIORITY: critical
TRAINING DURATION: 1h

NAME      STATUS   TRAINER  AGE  INSTANCE                 NODE
critical  RUNNING  MPIJOB   1h   critical-launcher-4jm85  192.168.168.124
critical  RUNNING  MPIJOB   1h   critical-worker-0        192.168.168.124
```

- scenario 4: if the priority is not set, the value of  PRIORITY is `NO SET`

```
arena get firstjob
STATUS: SUCCEEDED
NAMESPACE: default
PRIORITY: N/A
TRAINING DURATION: 49s

NAME      STATUS     TRAINER  AGE  INSTANCE          NODE
firstjob  SUCCEEDED  TFJOB    16h  firstjob-chief-0  192.168.168.125

Your tensorboard will be available on:
http://192.168.168.120:32196
```

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/arena/209)

### CODE REVIEW

File: cmd/arena/commands/get_advanced.go [hunk 1]
```diff
@@ -94,8 +94,21 @@ func BuildJobInfo(job TrainingJob) *types.JobInfo {
...
 	}
 }
+
+/**
+* getPriorityClass returns priority class name
+ */
+func getPriorityClass(job TrainingJob) string {
+	pc := job.GetPriorityClass()
+	if len(pc) == 0 {
+		pc = "NO SET"
```
Comment (wsxiaozhang) [inline cmd/arena/commands/get_advanced.go]: "Not Set" would be better

Comment (cheyang) [reply cmd/arena/commands/get_advanced.go]: Updated.

**Pull request merged**